### DISCUSSION TITLE
Original JWordNet source?

### DISCUSSION THREAD

**Discussion by xperia64** [nezda/yawni#7] [General]
Hi,
I've been attempting to locate the original JWordNet source by Oliver Steele, and was wondering if you still had copies of the original releases you could share.

While searching for "JWordNet.zip", I noticed a cached Google search description claiming the defunct yawni SourceForge page had downloads of at least JWordNet 1.0 and 1.1, but neither these nor the original jwn SourceForge page downloads were backed up on the internet archive.

Thanks.

**Comment by nezda** [comment]
Hi @xperia64 !

I've done some basic digging around but all of I've come up with so far is versions of https://github.com/nezda/yawni/commit/c059e9450d0e0345a50c93e5d1a24b9237352d0c which isn't what you asked for. If I have older copies, I'm afraid they're going to be quite challenging for me to find. 

I'm curious why you're looking for these versions: are you a big fan of his (same)? an archivist? hate these versions? This is the most interest I've seen in this project (naturally nothing to do with my work, heh) and I'll do some more work for you (and the history "books"). Worst case, he probably has copies but it'll be more fun / interesting to dig a bit.

**Comment by xperia64** [comment]
Hi,

Half-archivist, half that I'm working on some legacy Java NLP software which seems to have used the original JWordNet for something. To be honest I'm not completely sure what it was used for, but I'm attempting to minimize what I have to decompile later by gathering the sources for the various libraries used as I come across them.

I did play around with the Yawni Browser a bit and it did answer some questions about how WordNet may have been used.

Thanks!
